gpiozero/i2c_devices.py

```diff
@@ -0,0 +1,49 @@
+# vim: set fileencoding=utf-8:
+#
+# GPIO Zero: a library for controlling the Raspberry Pi's GPIO pins
+#
+# Copyright (c) 2016-2021 Dave Jones <dave@waveform.org.uk>
+# Copyright (c) 2020 Grzegorz Szymaszek <gszymaszek@short.pl>
+# Copyright (c) 2016-2019 Andrew Scheller <github@loowis.durge.org>
+# Copyright (c) 2016-2018 Ben Nuttall <ben@bennuttall.com>
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from math import trunc
+import time
```

Comment:
Looks like these two imports aren't used?

Reply:
They should be used. `trunc` is used on line 303. `time` is used on line 235 and line 281.

Reply:
Ah. I know now what you meant. These imports are for the second commit. It's fixed now.